README.md

```diff
@@ -4,84 +4,108 @@
 ## <a href="https://www.neuropac.info/">NeuroPAC</a>
 
 ## Table of Contents:
-- [Survey paper](#survey_paper)
+- [Event-based Vision Resources](#event-based-vision-resources)
```

Comment:
Something seems to be affecting the entire table of contents. Could you check and restore? thanks.